eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -117,12 +119,13 @@ public void setActiveEditorPart(IEditorPart partRef) {
         if (!(partRef.getEditorInput() instanceof IFileEditorInput)) {
             return;
         }
-
+        if (partRef == activeEditorPart) {
+            return;
+        }
```

Comment:
Complete, plus parameters with function which is responsible for knowing that we are post building the project.